### ISSUE TITLE
version v1.0.5 doesn't find 'assetDirectory' in config

### ISSUE THREAD

**Issue by misterbrownlee** [topcoat/topdoc#44]
Something happened to config parsing such that setting 'assetDirectory' stopped working.  It still works if passed via the command line flag.  It just silently fails 😿  which is sort of rude.

Workaround is to pass via command line flag, which still works.

**Comment by misterbrownlee** [comment]
ohai... I thought you asked this here?  This worked in 1.0.3:

```
{
  "source": "dist/reference/css",
  "destination": "dist/reference/docs",
  "clobber": true,
  "assetDirectory": "docs-template",
  "pugFile": "docs-template/template.pug"
}
```

**Comment by GarthDB** [comment]
Ok - I just made a test and it is totally failing, good first step.

```js
test.cb('should include assets from asset directory in rc file', t => {
  const destination = path.resolve(baseDestination, randomstring.generate());
  const expected = read(path.resolve(__dirname,
    'fixtures', 'project', 'passets', 'css', 'pass.css'));
  nixt()
  .cwd(cwd)
  .expect(() => {
    const anotherFile = read(path.resolve(destination, 'css', 'another.css'));
    t.is(expected, anotherFile);
  })
  .run(`topdoc -d ${destination}`)
  .end(t.end);
});
```

**Comment by misterbrownlee** [comment]
This duplicates #42, @GarthDB can pick which one to close 💀

**Comment by GarthDB** [comment]
Ok - @misterbrownlee fixed in v1.0.7 (beta)
